content/en/user-manual/assets/animation.md

```diff
@@ -6,6 +6,17 @@ position: 4
 
 An Animation asset is used to play a single animation on a 3D model. Animations are imported by uploading 3D scenes (such as FBX files) which contain animation data. The animation data is extracted from the uploaded file by the [asset pipeline][asset_pipeline] and a [Target Asset][target_asset] is created to use in game.
 
+## Animation Preview
+
+The inspector has a viewer that can be used to preview the animation with any Template or Model asset. This is useful if you have a single animation that can be applied to different character rigs.
+
+![Asset Inspector Preview][inspector-preview-gif]
+
+There is also a viewer in the [Anim State Graph Editor][anim-state-graph-editor] when you select a state in the graph. It will show all Entities in the scene using the graph where you can select one to preview.
```

Comment:
That last sentence is kinda hard to understand. Could it be rephrased somehow to make it clearer?

Reply:
```suggestion
There is also a viewer in the [Anim State Graph Editor][anim-state-graph-editor] when you select a state in the graph. When selected, all Entities in the scene that are using the graph will show in a list and can be selected for preview.
```

How's this?

Reply:
Much better. 👍